aten/src/ATen/templates/TensorDense.cpp

```diff
@@ -6,7 +6,7 @@ Scalar ${Tensor}::localScalar() {
   return Scalar(${to_at_type}(${THStorage}_get(${state,} THTensor_getStoragePtr(tensor), tensor->storage_offset())));
 }
 std::unique_ptr<Storage> ${Tensor}::storage() {
-  auto storage = THTensor_getStoragePtr(tensor);
+  Storage* storage = THTensor_getStoragePtr(tensor);
   THStorage_retain(storage);
-  return std::unique_ptr<Storage>(new ${Storage}(storage));
+  return std::unique_ptr<Storage>(storage);
```

Comment:
This is also wrong, for the same reasons as in https://github.com/pytorch/pytorch/pull/9772/files#r204907148